### PR TITLE
SE part wording revision.

### PR DESCRIPTION
**Pull request by tremes**
Comments and updates are welcome!! I think there is really room for further improvements.

### CODE REVIEW

File: api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java [hunk 1]
```diff
@@ -128,14 +128,14 @@ private static SeContainerInitializer findSeContainerInitializer() {
 
 
     /**
-     * All classes from the packages of the specified classes will be added to the set of bean classes for the synthetic bean archive.
+     * All classes from the packages of the specified classes will be added to the set of bean classes of the synthetic bean archive.
```
Comment (johnament) [inline api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java]: -1

File: api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java [hunk 2]
```diff
@@ -144,14 +144,14 @@ private static SeContainerInitializer findSeContainerInitializer() {
     public abstract SeContainerInitializer addPackages(Class<?>... packageClasses);
 
     /**
-     * Packages of the specified classes will be scanned and found classes will be added to the set of bean classes for the synthetic bean archive.*
+     * Packages of the specified classes will be scanned and found classes will be added to the set of bean classes of the synthetic bean archive.*
```
Comment (johnament) [inline api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java]: -1

File: api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java [hunk 3]
```diff
@@ -162,14 +162,14 @@ private static SeContainerInitializer findSeContainerInitializer() {
 
 
     /**
-     * All classes from the specified packages will be added to the set of bean classes for the synthetic bean archive.
+     * All classes from the specified packages will be added to the set of bean classes of the synthetic bean archive.
```
Comment (johnament) [inline api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java]: -1

File: api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java [hunk 4]
```diff
@@ -212,7 +212,7 @@ private static SeContainerInitializer findSeContainerInitializer() {
     public abstract SeContainerInitializer addExtensions(Class<? extends Extension>... extensions);
 
     /**
-     * Add interceptors classes to the list of enabled interceptors for a synthetic bean archive.
+     * Add interceptors classes to the list of enabled interceptors of the synthetic bean archive.
```
Comment (johnament) [inline api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java]: for the

File: api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java [hunk 5]
```diff
@@ -221,7 +221,7 @@ private static SeContainerInitializer findSeContainerInitializer() {
 
 
     /**
-     * Add decorators for a synthetic bean archive. Decorator classes are automatically added to the set of bean classes for the synthetic bean archive.
+     * Add decorators to the synthetic bean archive. Decorator classes are automatically added to the set of bean classes of the synthetic bean archive.
```
Comment (johnament) [inline api/src/main/java/javax/enterprise/inject/se/SeContainerInitializer.java]: for

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 6]
```diff
@@ -3,22 +3,22 @@
 == Bootstrapping a CDI container in Java SE
 
 In Java SE, the CDI container must be explicitly bootstrapped by the user.
-This is performed with the `SeContainerInitializer` abstract class and its static method `newInstance()`.
+This is performed by the `SeContainerInitializer` abstract class and its static method `newInstance()`.
 
 `SeContainerInitializer` is a service provider of the service `javax.enterprise.inject.se.SeContainerInitializer` declared in META-INF/services.
-It allows the configuration of the CDI container before its bootstrapping.
+This class enables user to configure the CDI container before it is bootstrapped.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: "allows a user"

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 7]
```diff
@@ -3,22 +3,22 @@
...
 The `SeContainerInitializer.initialize()` method bootstraps the container and returns a `SeContainer` instance.
 
-User can shutdown the container manually by calling the `close()` method on `SeContainer` or automatically since `SeContainer` extends `AutoCloseable` interface.
+User can shut down the container manually by calling the `close()` method on `SeContainer` or automatically using try-with-resources since `SeContainer` extends `AutoCloseable` interface.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: shutdown

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 8]
```diff
@@ -3,22 +3,22 @@
...
 
 === `SeContainerInitializer` class
 
-CDI Container can be configured and bootstrapped from `javax.enterprise.inject.se.SeContainerInitializer` abstract class.
+CDI container can be configured and bootstrapped from `javax.enterprise.inject.se.SeContainerInitializer` abstract class.
 
-A CDI implementation is required to provide an implementation of `SeContainerInitializer` declared as a service-provider.
-`newInstance()` static method uses java service-provider to obtain an implementation of `SeContainerInitializer` and return an instance of it.
+A CDI implementation is required to provide an implementation of `SeContainerInitializer` declared as a service provider.
+`newInstance()` static method uses Java service provider to obtain an implementation of `SeContainerInitializer` and return an instance of it.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: `ServiceProvider`

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 9]
```diff
@@ -51,39 +51,38 @@ public abstract class SeContainerInitializer {
...
+* `initialize()` bootstraps the container and returns a `SeContainer` as defined in <<se_container>>.
 
 
 
-Every invocation of the `SeContainerInitializer.initialize()` method returns a new `SeContainer instance.
-The _application context_ is started automatically by the container on start up.
-An implementation need not support calling `initialize()` multiple times on `SeContainerInitializer` without the previously returned `SeContainer` being shutdown.
+Every invocation of the `SeContainerInitializer.initialize()` method returns a new `SeContainer` instance.
+The application context is started automatically by the container on start up.
+An implementation does not support multiple calls of `SeContainerInitializer.initialize()` method when the `SeContainer` is running.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: need not.  this way we are not excluding an imp feature.

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 10]
```diff
@@ -51,39 +51,38 @@ public abstract class SeContainerInitializer {
...
 
 
 [[se_container]]
 
 === `SeContainer` interface
 
 
-The `javax.enterprise.inject.se.SeContainer` interface provides access to the `BeanManager` and programmatic lookup as defined in <<dynamic_lookup>>.
-`SeContainer` also extends `AutoCloseable`, so when dereferenced, it should shutdown automatically.
+The `javax.enterprise.inject.se.SeContainer` interface provides an access to the `BeanManager` and programmatic lookup as defined in <<dynamic_lookup>>.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: remove an.

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 11]
```diff
@@ -51,39 +51,38 @@ public abstract class SeContainerInitializer {
...
+`SeContainer` also extends `AutoCloseable`, so when dereferenced, it should shut down automatically.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: one word

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 12]
```diff
@@ -96,13 +95,13 @@ public interface SeContainer extends Instance<Object>,AutoCloseable {
 ----
 
 
-* `close()` method explicitly shutdown the container.
-If it is called and the container was already shutdown, it throws an `IllegalStateException`.
-* `isRunning()` method returns `true` if called before container shutdown and `false` after.
+* `close()` method explicitly shuts down the container.
+If it is called and the container was already shut down, it throws an `IllegalStateException`.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: one word.

File: spec/src/main/asciidoc/javase/bootstrap_se.asciidoc [hunk 13]
```diff
@@ -96,13 +95,13 @@ public interface SeContainer extends Instance<Object>,AutoCloseable {
...
+* `isRunning()` method returns `true` if called before container shuts down and `false` after.
 * `getBeanManager()` method returns the `BeanManager` (as defined in <<beanmanager>>) for the running container.
-if it is called and the container was already shutdown, it throws an `IllegalStateException`.
+If it is called and the container was already shut down, it throws an `IllegalStateException`.
```
Comment (johnament) [inline spec/src/main/asciidoc/javase/bootstrap_se.asciidoc]: one word

**Comment by tremes** [comment]
@johnament Thanks a lot for revision. I'll update.

**Comment by antoinesd** [comment]
LGTM

**Pull request merged**